app/Console/Commands/MigrateOrganizationCommand.php

```diff
@@ -0,0 +1,814 @@
+                    $this->databaseManager->rollBack();
+                    logger()->channel('migration')->error($exception);
+                    $this->error($exception->getMessage());
+                    continue;
+                }
+            }
+        } catch (Exception $exception) {
+            logger()->channel('migration')->error($exception);
+            $this->error($exception->getMessage());
+        }
+    }
+
+    /**
+     * Updates organization document link.
+     *
+     * @param $aidStreamOrganizationId
+     * @param $iatiOrganization
+     *
+     * @return void
+     */
+    public function updateOrganizationDocumentLinkUrl($aidStreamOrganizationId, $iatiOrganization): void
+    {
+        if (!empty($iatiOrganization) && !empty($aidStreamOrganizationId)) {
+            $documentLink = [];
+            $orgDocumentLinks = $iatiOrganization->document_link;
+
+            if ($orgDocumentLinks && count($orgDocumentLinks)) {
+                foreach ($orgDocumentLinks as $data) {
+                    $data['url'] = $this->replaceDocumentLinkUrl($data['url'], $iatiOrganization->id);
+                    $documentLink[] = $data;
+                }
+            }
+
+            $iatiOrganization->document_link = count($documentLink) ? $documentLink : null;
+            $iatiOrganization->timestamps = false;
+            $iatiOrganization->saveQuietly(['touch'=>false]);
+        }
+    }
+
+    /**
+     * Migrates Aid stream document data to iati document table.
+     *
+     * @param $aidstreamOrganizationId
+     * @param $iatiOrganization
+     * @param $migratedActivitiesLookupTable
+     *
+     * @return void
+     * @throws \JsonException
+     */
+    public function migrateDocuments($aidstreamOrganizationId, $iatiOrganization, $migratedActivitiesLookupTable): void
+    {
+        $aidStreamDocument = $this->db::connection('aidstream')->table('documents')
+                                      ->where('org_id', $aidstreamOrganizationId)
+                                      ->get();
+
+        if (count($aidStreamDocument)) {
+            $iatiDocuments = [];
+            foreach ($aidStreamDocument as $aidDocument) {
+                $iatiDocuments[] = [
+                    'activity_id'     => null,
+                    'activities'      => $this->fillActivitiesId(
+                        $aidDocument->activities,
+                        $migratedActivitiesLookupTable
+                    ),
+                    'organization_id' => $iatiOrganization->id,
+                    'filename'        => $aidDocument->filename,
+                    'extension'       => getFileNameExtension($aidDocument->filename),
+                    'size'            => $aidDocument->file_size,
+                    'created_at'      => $aidDocument->created_at,
+                    'updated_at'      => $aidDocument->updated_at,
+                ];
+            }
+
+            $this->documentService->insert($iatiDocuments);
```

Comment:
does this insert quietly, need to confirm that this will not change the activity updated at. May update the activity updated at if touch is being used.